fetch-publication/src/test/java/no/unit/nva/publication/fetch/FetchPublicationHandlerTest.java

```diff
@@ -143,19 +144,6 @@ public void handlerReturnsInternalServerErrorResponseOnUnexpectedException()
             MESSAGE_FOR_RUNTIME_EXCEPTIONS_HIDING_IMPLEMENTATION_DETAILS_TO_API_CLIENTS));
     }
 
-    @Test
-    @DisplayName("handler Returns BadGateway Response On Communication Problems")
-    public void handlerReturnsBadGatewayResponseOnCommunicationProblems()
```

Comment:
This exception is never thrown by DynamoDBService (old service) or the ResourceService (new service)